source/Calamari.Azure/Calamari.Azure.csproj

```diff
@@ -0,0 +1,106 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.0.0.0</Version>
+    <TargetFramework>net452</TargetFramework>
```

Comment:
This should be `net45` as this is our current minimum requirements for tentacle.